### ISSUE TITLE
Deploy new Packet c2.arm.large host to replace linux_aarch64_packet_net_2

### ISSUE THREAD

**Issue by nuclearsandwich** [ros2/build_farmer#232]
Aiming to retire the current host because of the recurrences of #177. 

Additionally, the c2.large.arm has fewer (but faster) cores than the c1.large.arm and supports the armv7 (armhf) instruction set.

**Comment by nuclearsandwich** [comment]
There is a new `linux-packet-arm` machine type on ci.ros2.org that is theoretically capable of both armhf and arm64 builds.

ARM64 [![Build Status](https://ci.ros2.org/buildStatus/icon?job=test_ci_linux-aarch64&build=6)](https://ci.ros2.org/job/test_ci_linux-aarch64/6/)

ARMHF [![Build Status](https://ci.ros2.org/buildStatus/icon?job=test_ci_linux-armhf&build=25)](https://ci.ros2.org/job/test_ci_linux-armhf/25/)

**Comment by nuclearsandwich** [comment]
Had an unfortunate first night deployed. Drive failures resulted in the filesystem system going into read-only mode without taking the host out of the pool.

```
20862.265658] sd 0:0:0:0: [sda] tag#13 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.265664] sd 0:0:0:0: [sda] tag#13 CDB: Write(10) 2a 00 11 a3 1b a0 00 00 08 00
[20862.265667] print_req_error: I/O error, dev sda, sector 295902112
[20862.271879] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 8916310 (offset 0 size 4096 starting block 36987765)
[20862.271894] Buffer I/O error on device sda3, logical block 35687284
[20862.278284] sd 0:0:0:0: [sda] tag#12 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.278301] sd 0:0:0:0: [sda] tag#12 CDB: Write(10) 2a 00 00 f4 e0 c0 00 00 18 00
[20862.278311] print_req_error: I/O error, dev sda, sector 16048320
[20862.284319] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 28314121 (offset 0 size 0 starting block 2006041)
[20862.284321] Buffer I/O error on device sda3, logical block 705560
[20862.290456] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 28314121 (offset 495616 size 8192 starting block 2006043)
[20862.290469] Buffer I/O error on device sda3, logical block 705561
[20862.296594] Buffer I/O error on device sda3, logical block 705562
[20862.302692] sd 0:0:0:0: [sda] tag#11 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.302695] sd 0:0:0:0: [sda] tag#11 CDB: Write(10) 2a 00 00 a6 58 58 00 00 18 00
[20862.302696] print_req_error: I/O error, dev sda, sector 10901592
[20862.308699] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 28314120 (offset 0 size 0 starting block 1362700)
[20862.308703] Buffer I/O error on device sda3, logical block 62219
[20862.314748] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 28314120 (offset 5292032 size 8192 starting block 1362702)
[20862.314758] Buffer I/O error on device sda3, logical block 62220
[20862.320807] Buffer I/O error on device sda3, logical block 62221
[20862.326823] sd 0:0:0:0: [sda] tag#10 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.326825] sd 0:0:0:0: [sda] tag#10 CDB: Write(10) 2a 00 36 a6 8b e8 00 00 08 00
[20862.326826] print_req_error: I/O error, dev sda, sector 916884456
[20862.332911] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 6553983 (offset 0 size 0 starting block 114610558)
[20862.332913] Buffer I/O error on device sda3, logical block 113310077
[20862.339311] sd 0:0:0:0: [sda] tag#9 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.339320] sd 0:0:0:0: [sda] tag#9 CDB: Write(10) 2a 00 36 a6 8b c0 00 00 18 00
[20862.339325] print_req_error: I/O error, dev sda, sector 916884416
[20862.339395] usb 1-1.1: reset high-speed USB device number 3 using xhci-hcd
[20862.345430] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 6553983 (offset 0 size 0 starting block 114610555)
[20862.345435] Buffer I/O error on device sda3, logical block 113310072
[20862.351784] Buffer I/O error on device sda3, logical block 113310073
[20862.358236] sd 0:0:0:0: [sda] tag#8 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.358253] sd 0:0:0:0: [sda] tag#8 CDB: Write(10) 2a 00 36 a2 cc 00 00 00 08 00
[20862.358263] print_req_error: I/O error, dev sda, sector 916638720
[20862.364405] EXT4-fs warning (device sda3): ext4_end_bio:323: I/O error 10 writing to inode 28317387 (offset 0 size 4096 starting block 114579841)
[20862.364470] sd 0:0:0:0: [sda] tag#14 FAILED Result: hostbyte=DID_OK driverbyte=DRIVER_TIMEOUT
[20862.364480] JBD2: Detected IO errors while flushing file data on sda3-8
[20862.364495] sd 0:0:0:0: [sda] tag#14 CDB: Write(10) 2a 00 1c 26 76 88 00 00 60 00
[20862.364504] print_req_error: I/O error, dev sda, sector 472282760
[20862.370690] Aborting journal on device sda3-8.
[20862.375786] EXT4-fs error (device sda3): ext4_journal_check_start:61: Detected aborted journal
[20862.380459] EXT4-fs error (device sda3): ext4_journal_check_start:61: Detected aborted journal
[20862.380460] EXT4-fs (sda3): Remounting filesystem read-only
```

**Comment by nuclearsandwich** [comment]
A new new host has been deployed. Hopefully it does better today.

**Comment by nuclearsandwich** [comment]
Deploying a new host hasn't resolved #177 but the new machine can run armhf and arm64. We'll keep it for now and I will find a way to further the investigation of #177.
